webapp/get_job_feed.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+
+from canonicalwebteam.http import CachedSession
+from html import unescape
+
+
+api_session = CachedSession(
+    fallback_cache_duration=300, file_cache_directory=".webcache"
+)
+
+
+base_url = "https://api.greenhouse.io/v1/boards/Canonical/jobs"
+
+
+def get_job_feed(endpoint):
```

Comment:
I don't think one-line functions are particularly helpful (or even 3-line functions in most cases). This is taking DRY too far - you're obfuscating useful logic for the sake of saving at most one line of duplication.